templates/OpenEdx/0/docker-compose.yml

```diff
@@ -0,0 +1,125 @@
+db:
+    image: mysql:5.7
+    labels:
+        io.rancher.sidekicks: dbConfig
+    environment:
+        MYSQL_ROOT_PASSWORD: password
+        MYSQL_DATABASE: edxapp
+        MYSQL_USER: edxapp001
+        MYSQL_PASSWORD: password
+    volumes:
+     - ${EDX_MYSQL_DATA_ROOT}:/var/lib/mysql
+    volume_driver: ${volume_driver}
+    
+dbConfig:
+    image: mysql:5.7
+    labels:
+        io.rancher.container.hostname_override: container_name
+        io.rancher.container.start_once: true
+    command: |
+      bash -c "sleep 2m; mysql -uroot -ppassword -h db -e \"create database edxapp_csmh;\""

```

Comment:
@borisno2 You could inject the rancher answers are environment variables while starting the container and use them in your command. Something like mysql -uroot -p${USER_ENTERED_PASSWORD}